saleor/order/utils.py

```diff
@@ -187,14 +187,11 @@ def recalculate_order_weight(order):
 def update_taxes_for_order_line(
```

Comment:
Is the name still valid? We have now logic with expiration which means, that we could call this method and we won't update anything.